### ISSUE TITLE
Anonimous user can link its account with OAuth provider

### ISSUE THREAD

**Issue by Perdolique** [Perdolique/perd#95]
If the user is already authenticated but doesn't have any third-party provider linked, we can suggest doing that to keep data access.

### Blocked by

-  #96

**Comment by Perdolique** [comment]
Superseded by #104, which now owns session-preserving Twitch account linking and explicit identity-conflict behavior as part of #716.
